### ISSUE TITLE
Use .mailcap

### ISSUE THREAD

**Issue by stevenjaycohen** [kieselsteini/cgo#18]
`fmt -w 80 $FILENAME | less`
Currently, you expect the filename to always be at the end, yes? If I am right that this is not possible with the current cgorc by default, then I'd like to request that you consider having all of the media handlers read .mailcap and using the variables there.

I worked around this limitation for now with the following shell script:
```
#!/bin/sh
fmt -w $1 $2 | less
```
But, since many people who will want to use your gopher client already have quite elaborate mailcap files, instead of adding the ability to your preferences, it would be more advantageous for this program to link into the standard mailcap definitions.

**Comment by stevenjaycohen** [comment]
Been thinking about this... It's probably best to separate this into 2 issues

1. The Media Handlers could be improved by allowing them to parse variables.
2. A preference could be added to toggle whether mailcap should be referenced.

After some consideration, I can come up with a few possible reasons that someone might want to assign different handlers to your gopher client than what they use in say mutt or w3m.

**Comment by kieselsteini** [comment]
You are totally right and I fixed this issue in my "better" gopher client and cgo replacement delve.
Take a look here https://github.com/kieselsteini/delve

I don't know if I really want to change this in cgo, as the code base is old and rusty. Delve has much cleaner code (in my opinion).

**Comment by stevenjaycohen** [comment]
I use gopher while logged in at sdf.org (where cgo is pre-installed). When I look for delve I find:

delve - Inspect the contents of a Xapian database
delv - DNS lookup and validation utility

So, it seems to be a crowded "namespace" for an application. However, I do have an idea about that.

With Explor**er** and Konquer**or** having been popular names for web clients, have you considered **Delver** for the gopher client?

I'm happy to put in a request over at sdf.org that they migrate to Delve from cgo. I do think they would find it less of a "hassle" if there were no namespace issues associated with the migration of tools.

**Comment by kieselsteini** [comment]
Good to know that there is a conflict with other programs. Perhaps I should do some kind of survey to find a good name :)
